daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -480,7 +480,7 @@ dapr --app-id app2 run -- php app2.php
 Dapr automatically takes the data sent on the publish request and wraps it in a CloudEvent 1.0 envelope.
 If you want to use your own custom CloudEvent, make sure to specify the content type as `application/cloudevents+json`.
 
-See info about content types [here](#Content-Types).
+See info about content types [here](#content-types) and [Cloud Events message format](https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-overview/#cloud-events-message-format)
```

Comment:
Inter-docs links should follow the Hugo shortcode convention: https://docs.dapr.io/contributing/contributing-docs/#referencing-other-pages

```suggestion
Read about content types [here](#content-types), and about the [Cloud Events message format]({{< ref "pubsub-overview.md#cloud-events-message-format" >}}).
```